app/utils.ts

```diff
@@ -71,6 +71,13 @@ export function isEmptyString(val: unknown): boolean {
   return typeof val === "string" && val.trim().length <= 0;
 }
 
+export const publishDateFormatter = new Intl.DateTimeFormat("en-US", {
```

Comment:
typically constants are caps, and the other constants in this file follow that convention.

```suggestion
export const PUBLISH_DATE_FROMATTER = new Intl.DateTimeFormat("en-US", {
```

Reply:
Done

Reply:
Not asking you to revert this, but just a small thing: this isn't really a string constant, which is what we usually use SCREAM_CASE for. It's more akin to a function (or I guess object with methods). Anyway, not a big deal, I do think this is more consistent. Just personal preference is that SCREAM_CASE is used for string constants shared throughout the app.